src/utils/deprecate.tsx

```diff
@@ -1,16 +1,17 @@
 /* eslint-disable no-console */
 import {useRef, useCallback} from 'react'
+declare var __DEV__: boolean
```

Comment:
Is there a better place for this?

Reply:
Not sure. I think this is fine for now 👍